### PR TITLE
Trash

### PR DESCRIPTION
**Pull request by RahullRawat**
- Add to trash
- Restore from trash
- Permanently delete from trash

### CODE REVIEW

File: src/pages/Trash/Trash.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+											></i>
+											<i
+												className="fa-solid fa-trash"
+												onClick={() => deleteNotesFromTrashHandler(note)}
+											></i>
+										</div>
+									</div>
+								</div>
+							);
+						})
```
Comment (Jay-Gadhiya) [inline src/pages/Trash/Trash.js:64]: you can intended code well

**Review by Jay-Gadhiya — APPROVED** [review]
- feature works fine
- notes card design in good manner
- UI looks good

File: src/reducer/NotesReducer.js [hunk 2]
```diff
@@ -59,6 +59,18 @@ const notesReducer = (state, action) => {
...
+		case "MOVE_TO_TRASH":
+			return {
+				...state,
+				trash: [...state.trash, action.payload],
+			};
+
+		case "DELETE_NOTE_FROM_TRASH":
+			return {
+				...state,
+				trash: state.trash.filter((note) => note._id !== action.payload._id),
```
Comment (Sudhanshusoni45) [inline src/reducer/NotesReducer.js:71]: You can destructure {payload} from action here as it is in used more then once

File: src/pages/Trash/Trash.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+									<div
+										className="note-footer sm-text"
+										style={{ backgroundColor: `${note.bgColor}` }}
+									>
+										<h6>{note.createdTime}</h6>
+										<h6 className="tag-bg">{note.tags}</h6>
+										<div className="note-footer-icons">
+											<i
+												class="fa-solid fa-arrow-up"
+												onClick={() => restoreFromTrash(note)}
```
Comment (Sudhanshusoni45) [inline src/pages/Trash/Trash.js:54]: Try and correct your code indentation.

**Review by Sudhanshusoni45 — APPROVED** [review]
You can remove your Create Note button as it is not serving any purpose in your UI 
Other than suggested:
1.UI looks great.
2.Neat and clean code

**Pull request merged**